playground/web_bindings/src/lib.rs

```diff
@@ -55,7 +55,7 @@ struct Transpile {
 pub fn transpile(source: &str, format: &str) -> Result<String, PlaygroundError> {
     let result = CONTEXT.with(|ctx| {
         let mut ctx = ctx.borrow_mut();
-        eval(source, &mut ctx, &OutputFormat::new(format))
+        eval_no_document(source, &mut ctx, &OutputFormat::new(format))
```

Comment:
Transpile is used for multiple options in the playground (Html rendered, Html iframe, LaTeX, Other output formats) but we would only want to remove the document when viewing "Html rendered" so I would suggest creating a new `transpile_no_doc` function or something along those lines

Reply:
Good catch, thanks. Will fix